### PR TITLE
Player: ActionChange -> ActionHandler

### PR DESCRIPTION
**Pull request by fig02**
Motivation: I see this system as a list of "event handlers" that listen for certain events or conditions to make a certain action function run.

I was already starting to name these functions as `_HandleX`, for example `Player_ActionChange_HandleRolling` in https://github.com/zeldaret/oot/pull/1949

I think it would be nice if this was just in the system name instead. 

This also would satisfy the current discussion here about keeping the function names consistent with the actions https://github.com/zeldaret/oot/pull/1949#discussion_r1580030873
I dont really like `Player_ActionChange_HandleRoll`, it sounds awkward to me. But if the "handle" part is in the system name itself `Player_ActionHandler_Roll` I am okay with it.

### CODE REVIEW

**Review by Dragorn421 — APPROVED** [review]

**Review by mzxrules — APPROVED** [review]

**Comment by cadmic** [comment]
Does it really "handle actions" though?

> Motivation: I see this system as a list of "event handlers" that listen for certain events or conditions to make a certain action function run.

Maybe `Player_EventHandler_Roll`?

**Comment by cadmic** [comment]
To clarify, I wonder if ActionHandler is too similar to "action func" (since these handle actions), but I don't feel strongly about it

**Comment by fig02** [comment]
The action functions are the implementations, and these handle responding to certain things to start certain actions.
A fully complete name could be something like `Player_ActionChangeHandler_` but im not sure the clunkyness/length is worth it. 

I think EventHandler is far too broad. If "events" were more of an established concept that had widespread use itd be fine, but I dont think it really makes sense to call it that here.

Basically I think the existence of `Player_Action_` is enough for it to be clear these arent action functions, along with the documentation comments. And the way its used doesnt really indicate that it is an action itself. `actionFunc` still exists and gets used like it does in every other actor, the pattern there is established.

I see and understand the concern but think its fine in this case.

**Comment by fig02** [comment]
Does anyone who already approved have any thoughts on the above? @Dragorn421 @AngheloAlf @mzxrules

**Comment by cadmic** [comment]
I do think "action change" captures the idea behind the system better than "action", and it fits better with the current `TryActionChangeList` `sActionChangeFuncs`. Otherwise `Player_ActionHandler_Roll` vs `Player_Action_Roll` seem way too similar.

What out merely `Player_ActionChange_Roll`? It's shorter and I don't think "handle" adds anything here

**Comment by cadmic** [comment]
I see, ignore that second bit then. I still feel like "change" is more specific than "handler" though and better communicates what this subsystem does

**Comment by cadmic** [comment]
I guess that could be because I'm used to the current name though. Again I don't feel too strongly about it, and you already have more than enough approvals, so feel free to ignore me

**Comment by fig02** [comment]
I find more value in specifying that its handling the flow of actions and would like the word to be in there somewhere. `Player_ActionChange_Roll` gives me the feeling that its actually doing the rolling and not just setting it up and changing. Which is the exact opposite to the problem you have apparently. 

Feel like we need other opinions at this point

**Comment by AngheloAlf** [comment]
I do like how ActionHandler sounds over ActionChange. Like those functions are just handling the actions.
I guess EventHandler could work, but that's even more broad, and like Fig said we would need to introduce the Event concept to Player

**Comment by Dragorn421** [comment]
tbh I don't mind any name I just trust fig's judgement here
